contrib/sssd.spec.in

```diff
@@ -8,7 +8,7 @@
 %endif
 
 # sysusers depends on presence of sssd user
-%if 0%{?rhel} >= 10
+%if 0%{?fedora} >= 10
```

Comment:
rhel>=10 || fedora>=40

Reply:
Ah, btw, I now recall I asked to put only 'rhel' thee for a time being because we don't yet build --with-sssd-user=sssd on Fedora.
So I was wrong asking to add this in Fedora rebase MR request.
But having it ready upstream won't hurt...

Reply:
I'm setting fedora so I can easily test it without deploying rhel-10. This will go away once I move it out of draft.

fedora>=40 should be set when all non-root changes are done, no?

Reply:
> fedora>=40 should be set when all non-root changes are done, no?

right